.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Our DockerFile uses `node:18.8.0-slim` which I think is based on alpine. Should we use the same environment for CI and container?

Reply:
this is not the same, in GitHub workflows this defines the underlying system VM its running on and not the docker image. That being said just noticed that they are deprecating 18.x and we should probably be specific on the version so will upgrade to use v22

Reply:
I understand that they are not the same, I was just wondering if we should have the same configuration between DockerFile and GitHub Actions, just for saving ourselves headaches in the future.

Reply:
don't have an option for alpine on GitHub as far as I can tell, hopefully once we move to gitlab runners we can make this exact